suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -11,6 +11,7 @@ import { deviceAuthenticityActions } from '@suite-common/device-authenticity';
 
 import { deviceActions } from './deviceActions';
 import { DiscoveryRootState, selectDiscoveryByDeviceState } from '../discovery/discoveryReducer';
+import { HIDDEN_DEVICE_ID } from './deviceConstants';
```

Comment:
I think we should rename this `HIDDEN_DEVICE_ID` to something that will reflect it's for portfolio only like `PORTFOLIO_DEVICE_ID` or like that, because just `HIDDEN` gives you zero context what it is good for

Reply:
`PORTFOLIO_TRACKER_DEVICE_ID` sounds good? Portfolio can be for connected device as well

Reply:
https://github.com/trezor/trezor-suite/pull/9694/commits/c9f9b0587b4ba3bea0645ae010e508e0e0911fc4